src/jats/importer/jats-parser-utils.ts

```diff
@@ -56,6 +56,14 @@ const updateNodeID = (
     }
     return
   }
+  if (node.type === schema.nodes.general_table_footnote) {
+    // @ts-ignore - while attrs are readonly, it is acceptable to change them when document is inactive and there is no view
+    node.attrs = {
+      ...node.attrs,
+      id: `GeneralTableFootnote:${uuidv4()}`,
+    }
+    return
+  }
```

Comment:
What's the purpose of this?

Reply:
On encoding any node that has no ID will not be iterated on

Reply:
can we make the id static like "_"? do we need it to be a uuid?

Reply:
since there might be multiple general_table_footnotes in a single document, not sure if having duplicate IDs would cause any issues in FE side
@IslamJMomani thoughts?

Reply:
It's fine, we can leave it as is.